app/controllers/DefermentController.scala

```diff
@@ -92,12 +92,14 @@ class DefermentController @Inject()(identify: IdentifierAction,
     }
   }
 
+  // TODO - needs to change once the feature swtich is taken out
   private[controllers] def getHeaderMessage(userAnswers: UserAnswers): String = {
     flowService.dutyType(userAnswers) match {
         case "vat" => "deferment.headingOnlyVAT"
         case "duty" => "deferment.headingDutyOnly"
         case "both" => "deferment.headingVATandDuty"
-      }
+        case _ => "deferment.headingDutyOnly"
```

Comment:
How is this related to your change? Could you remove it please.

Reply:
this is a temporary fix to have deferment page working when using the new flow to make a submission, in the next story this will need to be reworked